build.cake

```diff
@@ -57,17 +50,16 @@ var PACKAGE_SOURCE = new string[]
     };
 
 // Test Runners
-var NUNITLITE_RUNNER = "nunitlite-runner.exe";
+var NUNITLITE_RUNNER_DLL = "nunitlite-runner.dll";
 
 // Test Assemblies
 var FRAMEWORK_TESTS = "nunit.framework.tests.dll";
-var EXECUTABLE_NUNITLITE_TESTS = "nunitlite.tests.exe";
+var EXECUTABLE_NUNITLITE_TESTS_EXE = "nunitlite.tests.exe";
+var EXECUTABLE_NUNITLITE_TESTS_DLL = "nunitlite.tests.dll";
 
 // Packages
 var ZIP_PACKAGE = PACKAGE_DIR + "NUnit.Framework-" + packageVersion + ".zip";
 
-bool isDotNetCoreInstalled = false;
-
```

Comment:
This `packages` in no longer uses. 🙂

Reply:
(There are also some helper methods which are unused, but unrelated to this PR e.g. RunGitCommand. Might be worth a look while we're in the area, but it's a separate issue!)

Reply:
`packages` is on my list of cleanup activities :D Did I add `RunGitCommand`? If so it had a target at one point to test whether the repo could build after a `git clean -xfd`.